sql/core/src/test/scala/org/apache/spark/sql/DatasetBenchmark.scala

```diff
@@ -166,21 +207,33 @@ object DatasetBenchmark {
     val numChains = 10
 
     val benchmark = backToBackMap(spark, numRows, numChains)
-    val benchmark2 = backToBackFilter(spark, numRows, numChains)
-    val benchmark3 = aggregate(spark, numRows)
+    val benchmark2 = backToBackMapPrimitive(spark, numRows, numChains)
+    val benchmark3 = backToBackFilter(spark, numRows, numChains)
+    val benchmark4 = aggregate(spark, numRows)
 
     /*
-    Java HotSpot(TM) 64-Bit Server VM 1.8.0_60-b27 on Mac OS X 10.11.4
-    Intel(R) Core(TM) i7-4960HQ CPU @ 2.60GHz
```

Comment:
since the hardware is different, I think you should rerun the whole benchmark and update the result.